draft/2021-12-01-this-week-in-rust.md

```diff
@@ -14,6 +14,8 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ## Updates from Rust Community
 
+* [Black Hat Week (Black Hat Rust is out 🍾)](https://kerkour.com/black-hat-rust-week-2021/)
```

Comment:
Thanks for the link!

It should move under one of the sub-categories (`###` rather than `##`).

As it doesn't fit in any of the other categories, I'd guess we'd put it under "Miscellaneous", but @nellshamrell can probably decide when it's merged.

Reply:
Ho My bad, I didn't notice the `##` instead of `###`.